web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -890,18 +892,27 @@ $.extend(fixmystreet.set_up, {
                 fixmystreet.map.setCenter(
                     marker.geometry.getBounds().getCenterLonLat(),
                     fixmystreet.map.getNumZoomLevels() - 1 );
+                history.pushState({
+                    reportId: reportId,
+                    reportPageUrl: reportPageUrl,
+                    mapState: fixmystreet.maps.get_map_state()
+                }, null);
             }
             return;
         }
 
+        if (fixmystreet.page === 'reports' || fixmystreet.page === 'around') {
```

Comment:
Any reason not to have `my` here too? I imagine the same behaviour works the same there too.

Reply:
Oops, yes :)